### PR TITLE
WIP: SAST fixes

### PR DESCRIPTION
**Pull request by mmelko**

### CODE REVIEW

File: hawtio-system/src/main/java/io/hawt/util/introspect/support/ClassScanner.java
Comment (tadayosi) [inline hawtio-system/src/main/java/io/hawt/util/introspect/support/ClassScanner.java:1]: It seems OK to me. Let's not just comment out but remove them now.

**Review by tadayosi — APPROVED** [review]
One comment but otherwise it looks good to me.

**Comment by tadayosi** [comment]
@mmelko Also don't forget to change the title of the PR removing `WIP:` from it when you really think it's ready.

File: hawtio-system/src/main/java/io/hawt/util/introspect/support/ClassScanner.java
Comment (mmelko) [reply hawtio-system/src/main/java/io/hawt/util/introspect/support/ClassScanner.java:1]: done

**Pull request merged**